script/main_test.py

```diff
@@ -1,25 +1,57 @@
 import unittest
+import main
```

Comment:
Check https://www.guru99.com/learn-python-main-function-with-examples-understand-main.html to understand why you have this error now.
The solution is:
```python
if __name__ == "__main__":
    main()
```